src/ngraph/op/extractimagepatches.hpp

```diff
@@ -0,0 +1,117 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#pragma once
+
+#include "ngraph/op/op.hpp"
+
+namespace ngraph
+{
+    namespace op
+    {
+        namespace v3
+        {
+            typedef struct
```

Comment:
Why is this a struct instead of the four attributes?

Reply:
i was following interpolate and after reading the team's wiki I misunderstood and thought It is desired to make struct. 
Do you recommend to change it to remove struct? In that case I will update the code.

Reply:
Thanks i got rid of "struct" type of attribute.